mailbox/store/src/test/java/org/apache/james/mailbox/store/AbstractMessageIdManagerStorageTest.java

```diff
@@ -67,96 +69,96 @@
     protected abstract MessageIdManagerTestSystem createTestingData() throws Exception;
 
     public void setUp() throws Exception {
-        session = new MockMailboxSession(MailboxFixture.USER);
-        otherSession = new MockMailboxSession(MailboxFixture.OTHER_USER);
+        aliceSession = new MockMailboxSession(MailboxFixture.USER);
+        bobSession = new MockMailboxSession(MailboxFixture.OTHER_USER);
```

Comment:
why don't you change the Fixture then ?

Reply:
Fear for colateral dammages with hard coded constants in other tests?

Reply:
DONE